roles/vpn/templates/etc_openvpn_server.conf.j2

```diff
@@ -190,6 +190,7 @@ ifconfig-pool-persist ipp.txt
 ;push "redirect-gateway def1 bypass-dhcp"
 push "redirect-gateway def1"
 push "dhcp-option DNS 10.8.0.1"
+push "dhcp-option DNS 8.8.8.8"
```

Comment:
I'll leave google dns as comment into config.
Dnsmasq isn't required even for openvpn server.